python/grass/script/core.py

```diff
@@ -54,6 +54,40 @@
 class Popen(subprocess.Popen):
     _builtin_exts = {".com", ".exe", ".bat", ".cmd"}
 
+    class StdinWrapper:
+        """
+        Decodes bytes into str if writing failed and text mode was automatically set.
+
+        Remove for version 9
+        """
```

Comment:
Can we already note the deprecation and planned removal (like they do for Python docs)